### PR TITLE
Adds an actually GOOD gamemode (PvPvE, Meteors + Rev)

### PR DESCRIPTION
**Pull request by Ben10083**
After extensive research into top-level algorithms, I have determined an optimal gamemode that is estimated to significantly boost Aurorastation popularity above our rivals: SPLURT and CM.

PvPvE: Meteors + Rev

Revolution during a Meteor Storm? This level of soul can be dangerous, but I believe our server can handle it, if barely.

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

File: aurorastation.dme [hunk 1]
```diff
@@ -691,8 +691,9 @@
 #include "code\game\gamemodes\mixed\bughunt.dm"
 #include "code\game\gamemodes\mixed\conflict.dm"
 #include "code\game\gamemodes\mixed\crossfire.dm"
-#include "code\game\gamemodes\mixed\feeding.dm"
+#include "code\game\gamemodes\mixed\feeding.dm""
```
Comment (Generalcamo) [inline aurorastation.dme:694]:
```suggestion
#include "code\game\gamemodes\mixed\feeding.dm"
```

File: aurorastation.dme [hunk 2]
```diff
@@ -691,8 +691,9 @@
...
 #include "code\game\gamemodes\mixed\incursion.dm"
+#include "code\game\gamemodes\mixed\PvPvE.dm
```
Comment (Generalcamo) [inline aurorastation.dme:696]:
```suggestion
#include "code\game\gamemodes\mixed\PvPvE.dm"
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

**Comment by NonQueueingMatt** [comment]
Meteor gamemodes will not be accepted without an extensive rework of meteor

**Pull request closed without merging**